samtranslator/model/sam_resources.py

```diff
@@ -2282,16 +2283,18 @@ def _construct_appsync_schema(
         return schema
 
     def _construct_ddb_datasources(
-        self, ddb_datasources: Dict[str, aws_serverless_graphqlapi.DynamoDBDataSource], api_id: Intrinsicable[str]
-    ) -> List[DataSource]:
-        datasources: List[DataSource] = []
+        self,
+        ddb_datasources: Dict[str, aws_serverless_graphqlapi.DynamoDBDataSource],
+        api_id: Intrinsicable[str],
+        kwargs: Dict[str, Any],
```

Comment:
Caller says `**kwargs` is `Any`.

Reply:
I think when `kwargs` is unpacked with the `**` operator, it turns into `Dict[str, Any]`

Reply:
https://stackoverflow.com/questions/57897649/are-the-keys-of-a-kwargs-argument-to-python-function-guaranteed-to-be-type-strin